### DISCUSSION TITLE
Use `pip install ruff==<new version>` instead of `pip install ruff` in a release announcement blog post?

### DISCUSSION THREAD

**Discussion by harupy** [astral-sh/ruff#6668] [General]
<img width="873" alt="image" src="https://github.com/astral-sh/ruff/assets/17039389/c55bef14-d67c-47fa-833e-2ded378c1e35">

`pip install ruff` is a no-op if ruff is already installed but `pip install ruff==<release version>` is not.

**Comment by zanieb** [comment]
Thanks! I'll change the template to be `pip install --upgrade ruff`

https://github.com/astral-sh/astral-sh/pull/38
